website/docs/d/is_backup_policy_plans.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : is_backup_policy_plans"
+description: |-
+  Get information about backup policy plans.
+---
+
+# ibm_is_backup_policy_plans
+
+Provides a read-only data source for BackupPolicyPlanCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_backup_policy_plans" "example" {
+	backup_policy_id = "backup_policy_id"
+	name = "my-policy-plan"
+}
+```
+
+## Argument Reference
+Review the argument references that you can specify for your data source. 
+
+- `backup_policy_id` - (Required, string) The backup policy identifier.
+- `name` - (Optional, string) The unique user-defined name for this backup policy plan.
```

Comment:
name is not available in the api-spec, would we really want to show it, when name filter is not provided (coming blank ).